lib/core/auth.js

```diff
@@ -188,44 +193,63 @@ export default class Auth {
   // Token helpers
   // ---------------------------------------------------------------
 
-  getToken (strategy) {
-    const _key = this.options.token.prefix + strategy
+  getToken () {
```

Comment:
I was thinking why don't just make token public for explicit `$aurth.token.*` calls?

Reply:
Yeah why not!?

Reply:
We tried to keep it as close to the original as possible, but hey, the powers you get with a new major version! ✨

Reply:
Then we don't need wrapper functions like `getToken` here (it is `$auth.token.getToken()`)

Reply:
@pi0 We can't make this refactor. Me and @MathiasCiarlo has discussed about this and it will cause naming issue.
We can't move `$auth.refreshToken()` to `$auth.token.refresh()` since `$auth.refreshToken()` also refreshes the refresh token. Then we can't make the property `refreshToken` public because the name is already in use by `$auth.refreshToken()` method. That's why we kept the methods the the same way as before.

Reply:
Thinking about it now, another approach can be rename the method to `$auth.refreshTokens()` (plural). Then we can remove the wrapper functions! 😄

Reply:
What your thoughts?

Reply:
Refactored in commits https://github.com/nuxt-community/auth-module/pull/548/commits/76e322166097145c73f17ec91190dae2416cbda9 and https://github.com/nuxt-community/auth-module/pull/548/commits/558dfa7c5bb6f65e02b2449028ef1c656c5bfda2